mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -388,6 +388,32 @@ class LocalLDAModel private[spark] (
   }
 
   /**
+   * Predicts the topic mixture distribution for a document (often called "theta" in the
+   * literature).  Returns a vector of zeros for an empty document.
+   *
+   * Note this means to allow quick query for single document. For batch documents, please refer
+   * to [[topicDistributions(documents: RDD[(Long, Vector)])]] to avoid overhead.
```

Comment:
The Scala doc for this line is not generated correctly.  Can you try removing the argument and just writing ```[[topicDistributions]]``` instead?